tests/test_lib_caconnector.py

```diff
@@ -280,6 +281,25 @@ def test_05_templates(self):
         self.assertTrue(ddiff.days > 740, ddiff.days)
         self.assertTrue(ddiff.days < 760, ddiff.days)
 
+    def test_99_cleanup(self):
```

Comment:
Could we also reset the modified files `index.txt` and `serial`?

Reply:
will check

Reply:
I created a backup copy of the files and move them back in the cleanup step.
Move the cleanup step into teardownClass.

Reply:
done in 151a9d2d